### DISCUSSION TITLE
🚀 Feature Request: wrangler pages dev should restart when .dev.vars changes

### DISCUSSION THREAD

**Discussion by mariusGundersen** [cloudflare/workers-sdk#13517] [Feature Requests]
### Describe the solution

Changes made to the `.dev.vars` file while `wrangler pages dev` is running aren't picked up. This forces me to restart the dev server when making changes to environment variables. It would be nice if it could restart automatically, like it does when changes happen to the functions.

**Comment by CarmenPopoviciu** [comment]
hi @mariusGundersen , thank you for reporting this issue.

I'm triaging this as part of https://github.com/cloudflare/workers-sdk/milestone/10, and will discuss with the team if this is something we want to handle as part of that body of work
